### PR TITLE
Updated README.md

### LINKED ISSUE #32: update README.md
clone the project and install as per instructions then made following changes to file:
- update section **Command line options**.
- update all the **Examples** as per new options.
- update **how to install** section.

### PR DESCRIPTION
**Pull request by shitalmule04**
Fix #32

### CODE REVIEW

**Review by ganeshhubale — COMMENTED** [review]
@shitalmule04 
Make changes asap.

File: README.md [hunk 1]
```diff
@@ -23,92 +24,83 @@ Readit is command line bookmark manager. It is a command line utility to add, de
...
+  --help                     Show this message and exit.
+
 ```
 
 #### Examples
 
 1. **Add**  bookmarks:
 
-       $ readit -a https://www.google.co.in/ https://www.facebook.com/
+       $ readit -a https://www.google.co.in/
```
[the remark this answers is not in the document]
Comment (shitalmule04) [reply README.md]: @ganeshhubale, done.

**Comment by ganeshhubale** [comment]
@shitalmule04 
also add an example to **clear database** 
- Clear database: `readit -c` **or** `readit --clear`

**Comment by ganeshhubale** [comment]
@shitalmule04 
- Update **command line options**:
```
Usage: readit [OPTIONS] [INSERT]...

  It performs database operations as per arguments passed by user.

Options:
  -a, --add TEXT       Add URLs
  -d, --delete TEXT    Delete a URL
  -c, --clear TEXT...  Clear database
  -u, --update TEXT    Update a URL
  -v, --view TEXT...   Show URLs
  --help               Show this message and exit.
```

**Comment by ganeshhubale** [comment]
@shitalmule04 
update **Dependencies** table
- Scripting language : Python 2.7+

**Comment by shitalmule04** [comment]
@ganeshhubale, done

**Comment by shitalmule04** [comment]
@ganeshhubale, currently not showing **command line options** as you suggested once you modify the code, I'll modify that too.

**Comment by ganeshhubale** [comment]
@shitalmule04 
please update **command line options** too

**Comment by ganeshhubale** [comment]
@shitalmule04 
I have commented changes for **command line options**.
Please add to file.

**Comment by ganeshhubale** [comment]
@shitalmule04 
Thank you

**Pull request merged**